packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
It doesn't look like there are any console calls in this file, is this still needed?